activejob/test/cases/argument_serialization_test.rb

```diff
@@ -19,7 +21,7 @@ class ClassArgument; end
   end
 
   [ nil, 1, 1.0, 1_000_000_000_000_000_000_000,
-    "a", true, false, BigDecimal(5),
+    "a", true, false, # BigDecimal(5),
```

Comment:
I left this as a comment to it can easily be restored.

Reply:
Please don't. There's nothing worse than commented out code.